examples/tests/valid/schedule.w

```diff
@@ -0,0 +1,14 @@
+bring cloud;
+
+class Schedule {  
+  onTick(handler: inflight (): str) {
+    new cloud.Function(inflight () => {
+      handler();
+    });
+  }
+}
+
+let s = new Schedule();
+s.onTick(inflight (): str => {
+  return "foo";
+});
```

Comment:
Maybe add a `test`

Reply:
Testing this was less clear than expected because it's not possible to write the full SDK Schedule resource in Wing yet, so I rewrote this into a different example that still validates you can capture inflight closures in `doubler.w`